tests/functionaltests/Tests/CortanaTests.mm

```diff
@@ -0,0 +1,103 @@
+#include "UIKit/UIApplication.h"
+
+using namespace ABI::Windows::ApplicationModel::Activation;
+using namespace ABI::Windows::Media::SpeechRecognition;
+using namespace Microsoft::WRL;
+
+MOCK_CLASS(MockSpeechRecognitionResult,
+           public RuntimeClass<RuntimeClassFlags<WinRtClassicComMix>, ISpeechRecognitionResult, ISpeechRecognitionResult2> {
+               InspectableClass(RuntimeClass_Windows_Media_SpeechRecognition_SpeechRecognitionResult,
+                                BaseTrust); // claim to be the implementation for the real system RuntimeClass for SpeechRecognitionResult.
+
+           public:
+               MOCK_STDCALL_METHOD_1(get_Status);
+               MOCK_STDCALL_METHOD_1(get_Text);
+               MOCK_STDCALL_METHOD_1(get_Confidence);
+               MOCK_STDCALL_METHOD_1(get_SemanticInterpretation);
+               MOCK_STDCALL_METHOD_2(GetAlternates);
+               MOCK_STDCALL_METHOD_1(get_Constraint);
+               MOCK_STDCALL_METHOD_1(get_RulePath);
+               MOCK_STDCALL_METHOD_1(get_RawConfidence);
+               MOCK_STDCALL_METHOD_1(get_PhraseStartTime);
+               MOCK_STDCALL_METHOD_1(get_PhraseDuration);
+           });
+
+MOCK_CLASS(
+    MockVoiceCommandActivatedEventArgs,
+    public RuntimeClass<RuntimeClassFlags<WinRtClassicComMix>, IVoiceCommandActivatedEventArgs, IActivatedEventArgs> {
+        InspectableClass(RuntimeClass_Windows_ApplicationModel_Activation_VoiceCommandActivatedEventArgs,
+                         BaseTrust); // claim to be the implementation for the real system RuntimeClass for VoiceCommandActivatedEventArgs.
+
+    public:
+        MOCK_STDCALL_METHOD_1(get_Result);
+        MOCK_STDCALL_METHOD_1(get_Kind);
+        MOCK_STDCALL_METHOD_1(get_PreviousExecutionState);
+        MOCK_STDCALL_METHOD_1(get_SplashScreen);
+    });
+
+MOCK_CLASS(MockProtocolActivatedEventArgs,
+           public RuntimeClass<RuntimeClassFlags<WinRtClassicComMix>, IProtocolActivatedEventArgs, IActivatedEventArgs> {
+               InspectableClass(RuntimeClass_Windows_ApplicationModel_Activation_ProtocolActivatedEventArgs,
+                                BaseTrust); // claim to be the implementation for the real system RuntimeClass for Accel.
+
+           public:
+               MOCK_STDCALL_METHOD_1(get_Uri);
+               MOCK_STDCALL_METHOD_1(get_Kind);
+               MOCK_STDCALL_METHOD_1(get_PreviousExecutionState);
+               MOCK_STDCALL_METHOD_1(get_SplashScreen);
+           });
+
+// TODO: 8008558 Test handling foreground activation parameters from Cortana
+TEST(CortanaTest, VoiceCommandForegroundActivation) {
+    LOG_INFO("CortanaTest Voice Command Foreground Activation Test: ");
+    // Get current application to test on
+    WXApplication* app = [WXApplication current];
+    // Create mocked data to pass into application
+    auto fakeSpeechRecognitionResult = Make<MockSpeechRecognitionResult>();
+    fakeSpeechRecognitionResult->Setget_Text([](HSTRING* text) {
```

Comment:
nit: coding conventions state that comments on their own line should have a new line preceding them to clearly delineate with which line of code the comment belongs and to make the code more easily read in discrete blocks.